### PR TITLE
Purshottam_Assignment9

### PR DESCRIPTION
**Pull request by purshottam94**
image

### CODE REVIEW

**Review by surabhi-9 — CHANGES_REQUESTED** [review]
Rewrite the code again with proper validation like :
1. if packets are less than 0, show insufficient error
2. if requested packets are less than available packets then deduct requested quantity from finalQuantity else show message that we can give you packets (packets-totalQuantity )
and assign total packets to 0.

**Comment by purshottam94** [comment]
> Rewrite the code again with proper validation like :
> 
> 1. if packets are less than 0, show insufficient error
> 2. if requested packets are less than available packets then deduct requested quantity from finalQuantity else show message that we can give you packets (packets-totalQuantity )
>    and assign total packets to 0.

I have given validation inside displayOutofStockProduct and called this method inside purchaseProcess

File: src/purshottamJoshi/Shop.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	static int totalDosaStock = 43;
+	static int totalOilPouches = 39;
+	static int totalPanipuriPackets = 43;
+	static int totalMasalaPackets = 73;
+	void setCustomerDetails(String customer1) {
+		customerName = customer1;
+	}
+	void purchaseProcess(int maggie,int dosa,int oil,int panipuri,int masala) {
+		displayOutofStockProduct();
+		if(maggie >  totalMaggieStock) {
```
Comment (surabhi-9) [inline src/purshottamJoshi/Shop.java:15]: available packets will be when totalMaggieStock > maggie . so this condition is wrong

**Review by purshottam94 — COMMENTED** [review]
updated Purshottam_Assignment9 pr 
image
@surabhi-9 @harshvegada @KrishnaKTechnocredits

File: src/purshottamJoshi/Shop.java (hunk 1 above)
Comment (purshottam94) [reply src/purshottamJoshi/Shop.java]: > available packets will be when totalMaggieStock > maggie . so this condition is wrong

Updated same.

**Review by surabhi-9 — APPROVED** [review]
code looks good to me

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me!
Keep doing well!

**Pull request merged**